op-e2e/system_test.go

```diff
@@ -1344,6 +1347,180 @@ func TestWithdrawals(t *testing.T) {
 	require.Equal(t, withdrawAmount, diff)
 }
 
+// [Kroma: START]
+// TestKromaMptWithdrawals checks that a deposit and then withdrawal execution succeeds after Kroma MPT upgrade.
+// It verifies the balance changes on L1 and L2 and has to include gas fees in the balance checks.
+// It does not check that the withdrawal can be executed prior to the end of the finality period.
+func TestKromaMptWithdrawals(t *testing.T) {
```

Comment:
This functions seems to have lots of duplicated parts with `TestWithdrawals` function, so how about modifying to be like `TestFees`?